definitions/tools/subset_vcf_by_sample.cwl

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "subset a multi-sample vcf so that only a single sample is retained"
+baseCommand: ["/bin/bash", "subset_vcf.sh"]
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: 'zlskidmore/samtools:1.10'
```

Comment:
Maybe it's time for an updated version of `mgibio/samtools` (and, I suppose, `mgibio/samtools-cwl`)!

Reply:
I needed a container that had bcftools, bgzip, and tabix. This one fit the bill.  I suppose we could build one of our own, but in general, when one exists, then i take the path of least resistance